data_anal.py

```diff
@@ -0,0 +1,86 @@
+import glob
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from scipy import stats
+
+sns.set()
+
+
+class DataAnalysis:
+    def __init__(self):
+        self.final_results = []
+
+    def analise_data(self):
+        """
+        Runs sequence of methods. Clears data, sort it and makes a plot.
+        :param col: int
+        :return: None
+        """
+        self.read_data_for_analysis()
```

Comment:
Ta metoda jest zdefiniowana jako statyczna.
Ona owszem czyta dane i je zwraca, ale tutaj one przepadają.
Albo niech zostanie statyczna i łap tutaj te dane na zmienną albo niech przestanie być statyczna i korzysta z jakiegoś pola do tego

Reply:
Albo raczej.. czy to wywołanie jest tu potrzebne...?

Reply:
chyba nie jest, biorąc pod uwagę, że bez niej też działa
dobra to są jakieś artefakty, bo tutaj wykorzystałęm to:
    def split_data_for_analysis(self):
        self.final_results.extend([line.split(',') for line in self.read_data_for_analysis()])

Reply:
chociaz teraz tego w sumie troche nie czaje ;/
bo mam sobie split_data_for_analysis i on ma list comprehension, gdzie leci w pętli for po linijce zwroconej przez read_data_for_analysis, ale to w tym momencie wyglada jakby lecial tylko dla 1 pliku ;//

Reply:
czy split data wywoluje w tym momencie metode read data, gdzie jest petla i robi splita dla kazdego pliku?

Reply:
Huehuehue 
Nie zauważyłem tego.
W pętli for w metodzie, która czyta pliki masz return.
A return kończy działanie metody. W tym przypadku po przeczytaniu pierwszego pliku.